src/tests/java/htsjdk/samtools/SamReaderFactoryTest.java

```diff
@@ -132,8 +132,14 @@ public void samRecordFactoryTest(final String inputFile) throws IOException {
     public Object[][] composeAllPermutationsOfSamInputResource() {
         final List<SamInputResource> sources = new ArrayList<SamInputResource>();
         for (final InputResource.Type dataType : InputResource.Type.values()) {
+            if (dataType.equals(InputResource.Type.SRA_ACCESSION))
+                continue;
+
             sources.add(new SamInputResource(composeInputResourceForType(dataType, false)));
             for (final InputResource.Type indexType : InputResource.Type.values()) {
+                if (indexType.equals(InputResource.Type.SRA_ACCESSION))
+                    continue;
+
```

Comment:
That's fine, but you should add some tests involving SRA files to `SRATest` then (and check in the SRA files to the testdata directory)